cli/command/stack/remove_test.go

```diff
@@ -52,6 +54,40 @@ func TestRemoveStack(t *testing.T) {
 	assert.NoError(t, cmd.Execute())
 	assert.Equal(t, allServiceIDs, cli.removedServices)
 	assert.Equal(t, allNetworkIDs, cli.removedNetworks)
+	assert.Nil(t, cli.removedSecrets)
+	assert.Nil(t, cli.removedConfigs)
+
+	// Using API 1.25; removes services, networks, but doesn't remove configs
```

Comment:
I generally prefer to make these separate test cases instead of inline comments. It's never really clear where an inline comment ends. With a separate test function the scope is obvious.

Any shared setup can be moved into a helper function to remove the duplication, which also helps make the test more readable.

This is a minor style thing, so not blocking.